apps/test-app/cypress/integration/submission.ts

```diff
@@ -132,6 +132,17 @@ describe("Submission", () => {
     cy.findByText("Submitted!").should("exist");
   });
 
+  it("should submit using the formMethod of the submitter", () => {
+    cy.visit("submission/submitter");
+    cy.findByText("Submit GET").click();
+    cy.findByText("Submitting...").should("exist");
```

Comment:
I think this is happening too fast for the test to pick up, so it's not passing. Maybe if you add a timeout to the action?